tests/unit/client/feedback/dataset/remote/test_dataset.py

```diff
@@ -117,6 +117,9 @@ def create_mock_routes(
 
 
 class TestSuiteRemoteDataset:
+    def test_search_records(self) -> None:
```

Comment:
@alvarobartt apart from this missing test do you think there is another one that I need to implement?